packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseModal.tsx

```diff
@@ -9,10 +9,10 @@ import * as deviceUtils from '@suite-common/suite-utils';
 import {
     selectIsDiscoveryAuthConfirmationRequired,
     selectDevices,
+    onPassphraseSubmit,
 } from '@suite-common/wallet-core';
 
 import { useSelector, useDispatch } from 'src/hooks/suite';
-import { onPassphraseSubmit } from 'src/actions/suite/modalActions';
```

Comment:
I think you forgot to delete the original one.

Reply:
Good catch, thanks https://github.com/trezor/trezor-suite/pull/10738/commits/b4f72a2875ed302e2cc5a5a18ebe8afc4c87ad93